src/common/components/preferences/index.tsx

```diff
@@ -133,7 +147,7 @@ export class Preferences extends BaseComponent<Props, State> {
                 <Form.Label>{_t("preferences.notifications")}</Form.Label>
                 <Form.Control
                   type="text"
-                  value={global.notifications ? 1 : 0}
+                  value={ls.get("notifications") || global.notifications ? 1 : 0}
```

Comment:
Couldn’t we use api response instead local storage?

Reply:
yes.
Instead of updating the localStorage and then use the value of localStorage, i updated the store state and then use it.